### PR TITLE
doc(data-handling-overview): Update description and title

### PR DESCRIPTION
**Pull request by aibcmars**
Allow a better SEO

### CODE REVIEW

File: md/data-handling-overview.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Data handling overview
 
+How to effectively use the different kinds of data in the Bonita application
```
Comment (abirembaut) [inline md/data-handling-overview.md]: We usually talk about Bonita Platform in the rest of the documentation. So I would change the end of the sentence. What do you think of : "How to effectively use the different kinds of data available within Bonita Platform" ?

**Review by abirembaut — COMMENTED** [review]
Just a comment on the description

File: md/data-handling-overview.md (hunk 1 above)
Comment (aibcmars) [inline md/data-handling-overview.md]:
```suggestion
How to effectively use the different kinds of data available within Bonita Platform
```

Comment (aibcmars) [reply md/data-handling-overview.md]: I agree it should be consistent with other docs

**Pull request merged**